app/locales/fr.json

```diff
@@ -86,7 +86,8 @@
             "menubarIcon": "Icône de la bar de menu :",
             "colour": "Couleur",
             "monochrome": "Monochrome",
-            "invertedMonochrome": "Monochrome inversé"
+            "invertedMonochrome": "Monochrome inversé",
+      "snowWhite": "Snow white"
```

Comment:
now that we use Weblate for translations, it's actually better to leave untranslated strings out. So where strings are not translated yet, please remove them.

Reply:
done